packages/nocodb-sdk/src/lib/sqlUi/PgUi.ts

```diff
@@ -1436,7 +1436,7 @@ export class PgUi {
       case 'daterange':
         return 'string';
       case 'double precision':
-        return 'string';
+        return 'float';
```

Comment:
Please merge this case with float4, float8 as you've within ModelXcMetaPg.ts too keep it consistent.